src/main/java/com/starbucks/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.starbucks.exception;
+
+public class ResourceNotFoundException extends RuntimeException {
```

Comment:
You should add an annotation which returns the error code (in Todo something like ResponseCode) and put 404 which is the standard code for Not_Found exception. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status you should check it here